plugins/mac-avcapture/av-capture.mm

```diff
@@ -2209,34 +2218,48 @@ static void add_manual_properties(obs_properties_t *props)
 		OBS_COMBO_FORMAT_STRING);
 	obs_property_list_add_string(dev_list, "", "");
 
-	NSArray *devices = nil;
-
-	AVCaptureDeviceDiscoverySession *mediaDeviceDiscoverySession = [AVCaptureDeviceDiscoverySession
-		discoverySessionWithDeviceTypes:@[
-			AVCaptureDeviceTypeBuiltInWideAngleCamera,
-			AVCaptureDeviceTypeExternalUnknown
-		]
-				      mediaType:AVMediaTypeVideo
-				       position:AVCaptureDevicePositionUnspecified];
-	NSArray *mediaDevices = [mediaDeviceDiscoverySession devices];
-
-	AVCaptureDeviceDiscoverySession *muxedDeviceDiscoverySession = [AVCaptureDeviceDiscoverySession
-		discoverySessionWithDeviceTypes:@[
-			AVCaptureDeviceTypeExternalUnknown
-		]
-				      mediaType:AVMediaTypeMuxed
-				       position:AVCaptureDevicePositionUnspecified];
-	NSArray *muxedDevices = [muxedDeviceDiscoverySession devices];
-
-	devices = [mediaDevices arrayByAddingObjectsFromArray:muxedDevices];
-
-	for (AVCaptureDevice *dev in devices) {
-		if ([dev hasMediaType:AVMediaTypeVideo] ||
-		    [dev hasMediaType:AVMediaTypeMuxed]) {
+#if __MAC_OS_X_VERSION_MAX_ALLOWED >= 101500
+	if (__builtin_available(macOS 10.15, *)) {
```

Comment:
We require macOS 10.15 now, so these checks can safely be removed.

Reply:
Removed, thanks!